### PR TITLE
Inherit dot now positioned correctly

### PR DESCRIPTION
**Pull request by MaksymShokin**
@sofiiakvasnevska 

## Issue
https://github.com/Fliplet/fliplet-studio/issues/4870

## Description
Apparently the problem was that to some divs class 'multi-field' was not applied and the width of the container was too long. 

## Screenshots/screencasts
Before 
Before 2
Before 1

After
After 2
After 1

## Backward compatibility
This change is fully backward compatible.

### CODE REVIEW

**Comment by tonytlwu** [comment]
image

**Comment by MaksymShokin** [comment]
@tonytlwu fixed 
dot-demo

File: src/components/UI/InheritDot.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="inheritance-holder">
+  <div class="inheritance-holder" :class="{ 'inheritance-holder-left': colorPicker }">
```
Comment (tonytlwu) [inline src/components/UI/InheritDot.vue]: I'd rather have something like this. This way, the color field component will be responsible in dictating how the dot will be positioned, no the dot component being aware that "color picker" means "left aligned".

```suggestion
  <div class="inheritance-holder" :class="{ 'inheritance-holder-left': position === 'left' }">
```

File: src/components/fields/ColorField.vue [hunk 2]
```diff
@@ -5,7 +5,7 @@
         <div id="color-picker-container" class="color-holder" ref="colorSquare" :style="'background-color: ' + valueToShow" @click.prevent="toggleColorPicker"></div>
       </div>
       <div v-if="label" class="field-label" @click.prevent="toggleColorPicker">{{ label }}</div>
-      <inherit-dot v-if="!isInheriting" @trigger-inherit="inheritValue" :move-left="true" :inheriting-from="inheritingFrom"></inherit-dot>
+      <inherit-dot v-if="!isInheriting" @trigger-inherit="inheritValue" :color-picker="true" :move-left="true" :inheriting-from="inheritingFrom"></inherit-dot>
```
Comment (tonytlwu) [inline src/components/fields/ColorField.vue]:
```suggestion
      <inherit-dot v-if="!isInheriting" @trigger-inherit="inheritValue" :position="left" :move-left="true" :inheriting-from="inheritingFrom"></inherit-dot>
```

File: src/components/UI/InheritDot.vue (hunk 1 above)
Comment (MaksymShokin) [reply src/components/UI/InheritDot.vue]: fixed

File: src/components/fields/ColorField.vue (hunk 2 above)
Comment (MaksymShokin) [reply src/components/fields/ColorField.vue]: fixed

**Pull request merged**